.changeset/config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
We use changeset only to bump packages?

Reply:
yes, we only need to add a changeset before publish instead of in developing PR

Reply:
https://nx.dev/nx-api/nx/documents/release#dryrun seems promising, we could replace changeset in Rsbuild as well. IIRC, cc @chenjiahan wanted to replace the changeset before.

Reply:
The `web-infra-dev/actions@v2` does not support nx workflow, maybe we can use a lightweight bump and publish model write by ourselves👀

Reply:
Adding changeset by maintainer manually in our daily PR.
And bumping versions are operated in workflow

Reply:
But Rsbuild had stop adding changeset daily for a time being, and using changesets as a classic Lerna.

Reply:
Now they are publishing beta version and they just bump in local and make a PR now. Since former latest version, 
![image](https://github.com/user-attachments/assets/6b277447-ef34-4fb9-b351-853515a14326)
they add a changeset before publish or in daily development, just what I do in Rspress now.

Reply:
web-infra-dev/actions requires changesets now. We may look for the alternatives in the future.